addons/account_edi_ubl_cii/models/account_edi_xml_cii_facturx.py

```diff
@@ -166,6 +166,12 @@ def format_monetary(number, decimal_places=2):
         for line_vals in template_values['invoice_line_vals_list']:
             line = line_vals['line']
             line_vals['unece_uom_code'] = self._get_uom_unece_code(line)
+            # the gross_price_unit should be the price tax excluded !
+            # if price_unit = 100 and tax 10% price_include -> then the gross_price_unit is 91
+            gross_price_unit = line.price_unit
+            for tax in line.tax_ids.filtered("price_include"):
+                gross_price_unit = self.env.company.currency_id.round(gross_price_unit / (1 + tax.amount/100))
```

Comment:
I'm not quite sure about doing the rounding each time. It shouldn't happen often, but in case you have a 2-3 taxes price included on the line, don't you risk losing a cent here or there?